lib/facility_factory.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/facility'
+
+class FacilityFactory
+  def create_facilities(data)
+    data.map do |entry|
+
+    Facility.new(
+      name: entry[:dmv_office] || entry[:office_name],
```

Comment:
Great job getting the data from different sources to work! One thing you could think about is moving some of this data manipulation to a helper method.  You could imagine that all 50 states may have different data formats and that could make this method really overwhelming.  That refactor could look like:

```ruby
name: set_name_attribure(entry)

...

def set_name_attribute(facility)
  facility[:name] || facility[:dmv_office] || facility[:office_name] #etc...
end
```